hyperspy/tests/mva/test_bss.py

```diff
@@ -120,10 +120,14 @@ def test_diff_axes_string_with_mask(self):
         mask.fold()
         self.s.learning_results.factors[5, :] = np.nan
         factors = self.s.get_decomposition_factors().inav[:3]
+        mask.change_dtype("float")
+        mask_diff = mask.diff(axis="x", order=1)
+        mask.change_dtype("bool")
+        mask_diff.change_dtype("bool")
         self.s.blind_source_separation(
             3, diff_order=0, fun="exp", on_loadings=False,
             factors=factors.diff(axis="x", order=1),
-            mask=mask.diff(axis="x", order=1))
```

Comment:
I reverted this change and exclude numpy=1.13.0 in setup.py. Since the time between 1.13.0 and 1.13.1 was fairly short, this is very unlikely that someone got stuck with numpy 1.13.0!